tasks/serve.js

```diff
@@ -1,12 +1,13 @@
 const browsersync = require('metalsmith-browser-sync') // setup synchronised browser testing
 const metalsmith = require('../lib/metalsmith') // configured static site generator
 const paths = require('../config/paths.json') // specify paths to main working directories
-const generateServiceWorker = require('./build-service-worker') // setup service worker generation
+const generateServiceWorker = require('../lib/build-service-worker') // setup service worker generation
 
 // setup synchronised browser testing
 metalsmith.use(browsersync({
-  open: false,
+  open: true,
```

Comment:
What does this flag do, why is changed?

Reply:
It was originally true (in master). I have reverted it back to true here. It opens up a new browser window every time you `npm start`. This was becoming insanely annoying as I was testing so I disabled it.